libs/cases/frontend/view/src/lib/components/case-browse/case-browse.component.html

```diff
@@ -1,37 +1,87 @@
 <div class="absolute top-20 right-4 !z-[50] flex flex-row-reverse gap-x-4">
-    <ix-button [routerLink]="['/cases/create']" icon="plus">
-        Create Case
-    </ix-button>
-    <ix-button [routerLink]="['/cases/open']">
-        View Open Cases
-    </ix-button>
+	<ix-button [routerLink]="['/cases/create']" icon="plus"> Create Case </ix-button>
 </div>
+
 <ix-content>
-    <table class="table">
-        <thead>
-        <tr>
-            <th scope="col">#</th>
-            <th scope="col">Title</th>
-            <th scope="col">Priority</th>
-            <th scope="col">Status</th>
-            <th scope="col">Due to</th>
-            <th scope="col">Description</th>
-        </tr>
-        </thead>
-    <tbody>
-        @for (_case of _sortedCases(); track _case.id) {
-            <tr
-                [routerLink]="['/cases', _case.id]"
-                class="clickable-row"
-                [ngClass]="getStatusClasses(_case)"
-            >
-                <th scope="row">{{ _case.handle }}</th>
-                <td>{{ _case.title }}</td>
-                <td>{{ _case.priority }}</td>
-                <td>{{ _case.status }}</td>
-                <td>{{ _case.dueDate }}</td>
-                <td>{{ _case.description }}</td>
-            </tr>
-        }
-    </table>
+	<div class="pill-container-container">
+		<div class="pill-container">
+			<ix-pill (click)="flipShowStatusOpen()" [outline]="!showStatusOpen()">
+				open
+			</ix-pill>
```

Comment:
The <ix-pill> is according to siemens not supposed to be used like this. It should be possible using the Siemens iX [Toggle Button](https://ix.siemens.io/docs/controls/buttons/toggle-buttons) but might be annoying to make it look good